spec/factories/grda_warehouse/client_files.rb

```diff
@@ -1,10 +1,18 @@
 FactoryBot.define do
   factory :client_file, class: 'GrdaWarehouse::ClientFile' do
+    transient do
+      tags { [] }
+    end
+
     association :client, factory: :grda_warehouse_hud_client
     association :user, factory: :user
     client_file { Rack::Test::UploadedFile.new('spec/fixtures/files/images/test_photo.jpg', 'image/jpg') }
     name { 'Test File' }
     visible_in_window { true }
+
+    before(:create) do |file, evaluator|
+      file.tag_list = evaluator.tags.map(&:name)
+    end
```

Comment:
Nice. I am a big fan of the transient attributes. Much cleaner than the alternative (eg:
https://github.com/greenriver/hmis-warehouse/blob/e8f882c80e860b837b3acb99e7194d6ebdbcf53a/spec/models/grda_warehouse/client_file_legacy_spec.rb#L170-L177